src/gui/widgets/scrollbar_container.hpp

```diff
@@ -448,6 +448,10 @@ class scrollbar_container : public container_base
 	 */
 	virtual void handle_key_right_arrow(SDL_Keymod modifier, bool& handled);
 
+protected:
+	/** The builder needs to call us so we do our setup. */
+	void finalize_setup(); // FIXME make protected
```

Comment:
Since it's been made protected, there's no reason to keep the FIXME around.

Reply:
Removed.